cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -37,6 +43,21 @@ def from_model_id(cls, client: CogniteClient, data_model_id: dm.DataModelIdentif
         views = dm.ViewList(data_model.views)
         container_ids = views.referenced_containers()
         containers = client.data_modeling.containers.retrieve(list(container_ids))
+        while True:
+            referenced_containers = {
```

Comment:
Hard to read with double for-loop, this type of golfing is hard for readability.